src/main/resources/config/modules/Vanilla.xml

```diff
@@ -726,9 +726,9 @@
             <BiomeType name='Swamp'  />
             <BiomeType name='Beach'  />
             <BiomeType name='River'  />
-            <Biome name='Ocean'  weight='-1' />
-            <Biome name='Desert'  weight='-1' />
-            <Biome name='Mountain'  weight='-1' />
+            <Biome name='.*Ocean.*'  weight='-1' />
```

Comment:
These changes probably are not necessary since the distribution is already restricted to swamp, beach and river.